logging.js

```diff
@@ -3,6 +3,11 @@ const _isFiniteNumber = (val) => val !== null && isFinite(val) && !isNaN(val);
 
 const dataLoggingEndpointAttribute = 'data-logging-endpoint';
 const defaultThrottleRateMs = 60000;
+export const benignErrors = new Set([
+	'Script error.',
+	'ResizeObserver loop limit exceeded',
+	'ResizeObserver loop completed with undelivered notifications.'
```

Comment:
Does this cover the terminology for all the browsers, or are we maybe not filtering certain ones that are low on the list because of low usage (for example, Firefox's wording)?

Reply:
`ResizeObserver loop limit exceeded` is Chrome/Edge/Safari and `ResizeObserver loop completed with undelivered notifications.` is Firefox.